commands/stonks.js

```diff
@@ -27,15 +30,15 @@ Yesterday's Close: ${data.pc}\`\`\``);
     });
       
   } else {
-    message.reply(".stonks [ticker]")
+    message.reply(` \`Usage: ${message} [ticker]\``)
```

Comment:
Now replies with the command you used! Compatible with aliases and different prefixes